android/4_build_android_port.sh

```diff
@@ -32,14 +32,14 @@ git clone https://github.com/Ghabry/timidity_gus.git assets/timidity
 android update project --path "." --target 1
 
 # Build
-ndk-build -j4 NDK_LIBS_OUT=./jniLibs
```

Comment:
The script run on Linux and OSX.
Thanks for the urls, but the given solution doesn't seems to work on my OSX (and takes several minutes to terminate oO).